setup.py

```diff
@@ -2,6 +2,7 @@
 # where package_data is expected to be str and not unicode.
 from __future__ import absolute_import, division, print_function
 
+import os.path
```

Comment:
Haha interesting thread. Looks like it's not so obvious what to do :)

I'm guessing the fact that `os` injects `path` into `sys.modules` (and not done as a conditional import...) is a legacy artifact.